drools-ha/ha-core-infra/src/test/java/org/kie/hacep/LocalStorageKieSessionTest.java

```diff
@@ -127,7 +127,7 @@ public void deleteTest() throws ExecutionException, InterruptedException {
         assertEquals((Long) 0L, session.getFactCount().get());
     }
 
-    @Test(timeout = 10000)
+    @Test(timeout = 20000)
```

Comment:
I don't think  increasing timeout is good fix here

Reply:
Locally it spent 5 sec, on failure it spent 10.645 sec on pr build all six test in the class spent 24 sec and the fireuntil halt is under 10 sec and no failure, the execution time seems depend on the build execution.

Reply:
well there is some concurrent processing and this will be always very dependent on the environment, so I would like to see 1. eliminate concurrent access in Tests generally or 2. do not include such concurrency tests in default unit/integration run at all.

Reply:
3 Remove the timeouts because now are useless

Reply:
if that resolves the problem from opposite direction, why not

Reply:
Are you preparing follow up PR?

Reply:
Waiting the result of build 81 with this change, if green then a PR with LocalStorage without timeouts

Reply:
`test timed out after 20000 milliseconds` in [#82](https://rhba-jenkins.rhev-ci-vms.eng.rdu2.redhat.com/job/KIE/job/master/job/upstream-downstream/job/droolsjbpm-build-bootstrap.downstream/82/)